docs/mkdocs.yml

```diff
@@ -34,33 +34,56 @@ site_dir: 'generated'
 nav:
   - Welcome:
       - Landing: index.md 
-  - Overview:
-      - Readme: readme.md
-      - User guide: direct/user-guide.md
+  - What is KubeStellar?:
+      - Overview: readme.md
+#      - User guide: direct/user-guide.md
       - Architecture: direct/architecture.md
+      - Related:
+        - KubeFlex: direct/kubeflex-intro.md
+        - KubeStellar Galaxy: direct/galaxy-intro.md
       - Release-notes: direct/release-notes.md
-  - Getting Started: 
-          - Pre-reqs: direct/pre-reqs.md
+  - Using Kubestellar:
+          - Overview: direct/setup-overview.md
```

Comment:
It seems wrong to have two pages titled "Overview". Maybe this second one is "Overview of setting up KubeStellar"? or "Overview of KubeStellar setup"? Maybe "install" or "deploy" instead of "setup"?

Reply:
I was deliberately avoiding "deploy" for some of the same reasons you raise regarding workloads in your other comment.
How about "Basics" as the intro header?

Reply:
changed overview to basics